tests/unit/test_mongod_helpers.py

```diff
@@ -116,3 +121,76 @@ def test_is_replica_set_is_not_replica_returns_false(self, is_ready, close, find
         replica_set_status = mongo.is_replica_set()
         close.assert_called()
         self.assertEqual(replica_set_status, False)
+
+    @patch("mongod_helpers.MongoDB.client")
+    @patch("pymongo.MongoClient")
+    def test_reconfiguring_replica_invokes_admin_command(self, mock_client, client):
+        # presets
+        config = MONGO_CONFIG.copy()
+        mongo = MongoDB(config)
+        client.return_value = mock_client
+
+        # verify we make a call to replSetReconfig
+        mongo.reconfigure_replica_set()
+        mock_client.admin.command.assert_called()
+        command, _ = mock_client.admin.command.call_args
+        self.assertEqual("replSetReconfig", command[0])
```

Comment:
If only `command[0]` was needed, perhaps `(command, _), _` or similar is possible above?